loader/icd.c

```diff
@@ -324,14 +324,12 @@ void khrIcdLayerAdd(const char *libraryName)
 
     cl_uint limit = layerDispatchNumEntries < loaderDispatchNumEntries ? layerDispatchNumEntries : loaderDispatchNumEntries;
 
-    for (cl_uint i = 0; i < limit; i++) {
+    for (cl_uint i = 0; i < limit; i++)
```

Comment:
Minor: I'd prefer to keep the `{` curly braces `}`, even though these are single-line for loops.  This will avoid possible problems if additional lines are added in the future.